File: a20241107a(未整合的校正腳本)/plotCalbPoint3d_noText.py
import matplotlib.pyplot as plt
from mpl_toolkits.mplot3d import Axes3D
import numpy as np

def plotCalbPoint3d_noText(xy):
    num = len(xy)
    y = xy[:, 0]
    x = -xy[:, 1]
    z = xy[:, 2]
    fig = plt.figure()
    ax = fig.add_subplot(111, projection='3d')
    ax.scatter(x, y, z, marker='+')
    xmin = np.min(x)
    xmax = np.max(x)
    ymin = np.min(y)
    ymax = np.max(y)
    zmin = np.min(z)
    zmax = np.max(z)
    ax.set_xlim([xmin - 5, xmax + 5])
    ax.set_ylim([ymin - 5, ymax + 5])
    ax.set_zlim([zmin - 5, zmax + 5])
    ax.grid(True)
    # 無文字標籤
    ax.set_xlabel('Y')
    ax.set_ylabel('X')
    ax.set_zlabel('Z')
    plt.show()

File: a20241107a/plotCalbPoint3d_noText.m
function  plotCalbPoint3d(xy)
num=length(xy);
y=xy(:,1);x=-xy(:,2);z=xy(:,3);
plot3(x,y,z,'+');
xmin=min(x);xmax=max(x);
ymin=min(y);ymax=max(y);
zmin=min(z);zmax=max(z);
axis([xmin-5 xmax+5 ymin-5 ymax+5 zmin-5 zmax+5])

grid on
for n=1:num
    sn=num2str(n);
    text(x(n),y(n),z(n),sn)
end

xlabel('Y'),ylabel('X'),zlabel('Z')